scripts/normalize_grade_data.py

```diff
@@ -38,9 +38,12 @@
 data.CourseId = data.Course + '-' + data.Number.astype(str)
 
 # create year field from the file name
-data.Year = file[:4]
+#data.Year = file[:4]
+r = re.findall(r"([0-9][0-9][0-9][0-9])", file)
```

Comment:
This way it will find the year no matter where it is in the name of the file, not just the first 4 characters. When I ran it through tabula it put tabula- in the front of the original file name, and the code as it was did not like that haha.

Reply:
fair enough